engines/bladerunner/bladerunner.h

```diff
@@ -126,12 +137,19 @@ class BladeRunnerEngine : public Engine {
 	EndCredits         *_endCredits;
 	ESPER              *_esper;
 	GameFlags          *_gameFlags;
+	#if BLADERUNNER_RESTORED_CONTENT_GAME
```

Comment:
What is the restored content?

Reply:
This macro (remnant of old code of mine) has been removed